components/cards/default/card.hbs

```diff
@@ -18,7 +18,7 @@
     <div class="content-block__lower">
       <p class="content-block__subtitle">Digitalks Webinar</p>
       <h4 class="content-block__title">
-        <a href="#" class="anchor-primary">RSVP: This is a featured event that is topical and relevant.</a>
+        <a href="#" class="anchor-primary content-block--clickable">RSVP: This is a featured event that is topical and relevant.</a>
```

Comment:
If we want to leave this here rather than moving it up a parent element wrapper, let's not name it as a modifier to the root block. Instead, let's make it an element of the block like `content-block__link` or something like that.